velox/exec/Driver.cpp

```diff
@@ -108,43 +108,8 @@ velox::memory::MemoryPool* DriverCtx::addOperatorPool(
 
 std::optional<common::SpillConfig> DriverCtx::makeSpillConfig(
     int32_t operatorId) const {
-  const auto& queryConfig = task->queryCtx()->queryConfig();
```

Comment:
Another option would be to keep it here and lazy initialize the spill component of outputbuffer in partitioned output operator.